sass/includes/_section-separator.scss

```diff
@@ -0,0 +1,13 @@
+.section-separator {
+  width: 100vw;
+  position: relative;
+  left: calc(-50vw + 50%);
+
+  &__heading {
+    font-size: 2.5rem;
+    color: rgb(38, 38, 42);
+    text-align: center;
+    margin-top: 0px;
```

Comment:
I don't think you need the `px` qualifier here. `0` should suffice - and avoids the whole `rem` thing.